config/prometheus/collector_alerts.yaml

```diff
@@ -45,16 +45,6 @@ spec:
       labels:
         service: collector
         severity: critical
-    - alert: FluentdQueueLengthIncreasing
-      annotations:
-        message: "For the last hour, fluentd {{ $labels.pod }} output '{{ $labels.plugin_id }}' average buffer queue length has increased continuously."
-        summary: "Fluentd pod {{ $labels.pod }} is unable to keep up with traffic over time for forwarder output {{ $labels.plugin_id }}."
-      expr: |
-        sum by (pod,plugin_id) ( 0 * (deriv(fluentd_output_status_emit_records[1m] offset 1h)))  + on(pod,plugin_id)  ( deriv(fluentd_output_status_buffer_queue_length[10m]) > 0 and delta(fluentd_output_status_buffer_queue_length[1h]) > 1 )
-      for: 1h
-      labels:
-        service: collector
-        severity: Warning
     - alert: ElasticsearchDeprecation
       annotations:
         message: "The OpenShift Elasticsearch Operator is deprecated and is planned to be removed in a future release. Red Hat provides bug fixes and support for this feature during the current release lifecycle, but this feature no longer receives enhancements. As an alternative to using the OpenShift Elasticsearch Operator to manage the default log storage, you can use the Loki Operator."
```

Comment:
We should update this statement since THIS is the release where we are removing it.

Reply:
Yes, this part that i want to ask, should i remove this in this PR

Reply:
Yes, i want ask about that